app/scripts/modules/ecs/src/serverGroup/configure/wizard/capacityProvider/CapacityProvider.tsx

```diff
@@ -61,6 +89,24 @@ class EcsCapacityProvider extends React.Component<IEcsCapacityProviderProps, IEc
     this.setState({ capacityProviderStrategy: capacityProviderStrategy });
   };
 
+  private updateCapacityProviderStrategy = (targetCapacityProviderType: string) => {
```

Comment:
It feels like this method is doing a lot. Can we separate this into an update useDefaultCapacityProviders method and a updateCapacityProviderStrategy method (you can involve updateCapacityProviderStrategy from within useDefaultCapacityProviders, but it feels like this method has too much responsibility